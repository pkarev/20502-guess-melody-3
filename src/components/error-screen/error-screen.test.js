import React from 'react';
import renderer from 'react-test-renderer';
import ErrorScreen from './error-screen';

it(`Render ErrorScreen`, () => {
  const tree = renderer
  .create(<ErrorScreen/>)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
