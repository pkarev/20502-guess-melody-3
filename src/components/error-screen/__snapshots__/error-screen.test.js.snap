// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ErrorScreen 1`] = `
<section>
  <p>
    Произошла ошибка загрузки данных!!!
  </p>
  <p>
    Попробуйте перезагрузить страницу.
  </p>
  <p>
    В случае повторной ошибки попробуйте повторить попытку позднее
  </p>
</section>
`;
